### DISCUSSION TITLE
Is there a way to use a custom component as a selector in the html without using dyn-form?

### DISCUSSION THREAD

**Discussion by pookdeveloper** [myndpm/open-source#50] [Ideas]
Is there a way to use a custom component as a selector in the html without using dyn-form?
And it can't work with the component's own @INputs and @Outpus either, right?
Thanks!

**Comment by matheo** [comment]
There are internal dependencies that are not fulfilled without the `dyn-form` component like some services (`DynLogger`, `DynFormFactory`, `DynFormHandlers`, `DynFormResolver`) and tokens (`DYN_MODE`).

More importantly, the node service (`DynControlNode`) initialization relies in a complete hierarchy built according to the config.

It is not easy to build all the features offered by this lib with that kind of "independent" controls as leafs.
